shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/ClusterContextManagerCoordinatorTest.java

```diff
@@ -351,6 +356,39 @@ public void assertRenewInstanceOnlineEvent() {
         assertThat(((LinkedList<ComputeNodeInstance>) contextManager.getInstanceContext().getComputeNodeInstances()).get(1).getInstanceDefinition(), is(instanceDefinition1));
     }
     
+    @Test
+    public void assertCompleteUnitShowProcessList() {
+        final String showProcessListId = "show_process_list_id_1";
```

Comment:
What is the mean of `show_process_list_id_1`,  how about change it to `foo_process_id`?